Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "reflector-oracle"
-version = "4.0.0"
+version = "5.0.0"
```

It looks lie a minor change since the public interface or data reporting format was not changed.
```suggestion
version = "4.1.0"
```